<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddArmyFormClass</class>
 <widget class="QDialog" name="AddArmyFormClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>936</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AddArmyForm</string>
  </property>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>931</width>
     <height>491</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="1" column="5">
     <widget class="QLineEdit" name="ArmyNameLineEdit"/>
    </item>
    <item row="0" column="3" colspan="2" alignment="Qt::AlignmentFlag::AlignHCenter">
     <widget class="QLabel" name="label_1">
      <property name="text">
       <string>Available units</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Enter army name:&lt;b style =&quot;color:red;&quot;&gt;*&lt;/b&gt;</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" alignment="Qt::AlignmentFlag::AlignHCenter">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Armies</string>
      </property>
     </widget>
    </item>
    <item row="2" column="5" rowspan="7">
     <widget class="QTableWidget" name="ArmyUnitsTable">
      <column>
       <property name="text">
        <string>Units</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Amount</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="1" column="3" rowspan="8" colspan="2">
     <widget class="QListWidget" name="UnitsListWidget"/>
    </item>
    <item row="1" column="0" rowspan="8">
     <widget class="QListWidget" name="ArmiesListWidget"/>
    </item>
    <item row="0" column="7" rowspan="9">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="ToMenu">
        <property name="styleSheet">
         <string notr="true">background-color:#F0FFFF;
font-weight:800;
color:black;
</string>
        </property>
        <property name="text">
         <string>Go back</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="AddFortificationButton">
        <property name="text">
         <string>Add fortification</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="DeleteFortificationButton">
        <property name="text">
         <string>Delete fortification</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="AddArmyButton">
        <property name="text">
         <string>Add Army</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="DeleteArmyButton">
        <property name="text">
         <string>Delete Army</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="AddUnitButton">
        <property name="text">
         <string>Add unit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="DeleteUnitButton">
        <property name="text">
         <string>Delete unit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="6">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Army fortification:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="6">
     <widget class="QLabel" name="fortification">
      <property name="text">
       <string>default</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
